python/lsst/sconsUtils/scripts.py

```diff
@@ -312,6 +312,25 @@ def python(swigNameList=None, libs="main python", swigSrc=None):
         return result
 
     ##
+    #  @brief Convenience function to replace standard python/*/SConscript boilerplate.
+    #
+    #  @param nameList    Sequence of pybind11 modules to be built (does not include the file extensions).
+    #  @param libs        Libraries to link against, either as a string argument to be passed to
+    #                     env.getLibs() or a sequence of actual libraries to pass in.
```

Comment:
Please document what this function returns (and if you are willing to do that for other functions in this file, that would be great but I realize it is out of scope)